extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/pom.xml

```diff
@@ -94,6 +94,11 @@
             <artifactId>quarkus-jaxrs-client-reactive-deployment</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Are you sure this works properly? Just want to make sure it doesn't create any cyclic dependencies or anything

Reply:
Well, it's a test dep, and it seems to build… :shrug:

Reply:
OK :)